README.md

```diff
@@ -241,6 +245,34 @@ clerk.smsMessages
 You also consult the [examples folder](https://github.com/clerkinc/clerk-sdk-node/tree/main/examples) for further hints
 on usage.
 
+### Allowlist operations
+
+Allowlist operations are exposed by the `allowlistIdentifiers` sub-api (`clerk.allowlistIdentifiers`).
+
+#### getAllowlistIdentifierList()
+
+Retrieves the list of allowlist identifiers.
+
+```ts
+const allowlistIdentifiers = await clerk.allowlistIdentifiers.getAllowlistIdentifierList();
+```
+
+#### createAllowlistIdentifier(params)
```

Comment:
🙃 
Can we show the `CreateParams` type in this section ?

Also do you think it would be helpful to show a code snippet like:
```ts
await allowlistIdentifiers.createAllowlistIdentifier({
  identifier: 'test@example.com',
  notify: false,
});
```
after the params specification ?

WDYT?

Reply:
Forgot to add code snippets for create and delete. Added now. :smile: